proprietary/tokens/src/components/amsterdam/unordered-list.tokens.json

```
@@ -0,0 +1,22 @@
+{
+  "amsterdam": {
+    "unordered-list": {
+      "color": { "value": "{amsterdam.color.primary-black}" },
+      "font-family": { "value": "{amsterdam.typography.font-family}" },
+      "font-size": {
+        "narrow": { "value": "{amsterdam.typography.text-level.6.font-size.narrow}" },
+        "wide": { "value": "{amsterdam.typography.text-level.6.font-size.wide}" }
+      },
+      "font-weight": { "value": "{amsterdam.typography.font-weight.normal}" },
+      "line-height": { "value": "{amsterdam.typography.text-level.6.line-height}" },
+      "list-style": { "value": "square" },
+      "margin-block": { "value": "0.5rem" },
+      "padding-inline-end": { "value": "11px" },
+      "item": {
```

This naming (amsterdam-unordered-list-item-list-style-type) is a bit odd. Maybe we should add `level-1` (with `margin-block` and `list-style-type`) and `level-2` (with the same)?

---

Like a variant?


```suggestion
      "amsterdam": {
    	"unordered-list": {
			"level-1": {

			},
			"level-2": {
				
			}
		}
	  }
```

NLDS doesn't seem to be doing this.
see: utrecht/proprietary/design-tokens/src/component/utrecht/unordered-list.tokens.json

---

Hm, maybe the variant approach isn't quite correct either. But `item` isn't either, you're using it to style a `ul`, not `li`. Maybe something like this?

```
"amsterdam": {
  "unordered-list": {
    "unordered-list": {
      "list-style-type"
      etc.
    }
  }
}
```

---

`"item"` does give you a logical var name, one that resembles the component name `--amsterdam-unordered-list-item-list-style-type`

---

`--amsterdam-unordered-list-unordered-list-list-style-type` seems a bit excessive.